BricksAndHearts.UnitTests/ControllerTests/ControllerTestsBase.cs

```diff
@@ -25,12 +32,44 @@ protected BricksAndHeartsUser CreateUnregisteredUserInController(Controller _und
         };
 
         var unregisteredUser = new BricksAndHeartsUser(userDbModel, new List<Claim>(), "google");
-        var unregisteredUserPrincipal = new ClaimsPrincipal(unregisteredUser);
+        return unregisteredUser;
+    }
+    
+    protected BricksAndHeartsUser CreateAdminUser()
+    {
+        var userDbModel = new UserDbModel()
+        {
+            Id = 1,
+            GoogleUserName = "John Doe",
+            GoogleEmail = "test.email@gmail.com",
+            IsAdmin = true,
+            LandlordId = null,
```

Comment:
Some trailing commas in this file 🙂

Reply:
They cleared up when I rebased (I removed the trailing commas in the branch I merged yesterday, and made sure to take that version if the file when I rebased this branch).